### PR TITLE
Order show: Closes #47

### LINKED ISSUE #47: 13: Viewing a past order
Background: An existing user that has one previous order

As an authenticated user
When I visit "/orders"
Then I should see my past order
And I should see a link to view that order
And when I click that link
Then I should see each item that was order with the quantity and line-item subtotals
And I should see links to each item's show page
And I should see the current status of the order **(ordered, paid, cancelled, completed)**
And I should see the total price for the order
And I should see the date/time that the order was submitted

If the order was completed or cancelled
Then I should see a timestamp when the action took place

### PR DESCRIPTION
**Pull request by jimszalew**
created join table for orders and items
built individual order show page

### CODE REVIEW

**Comment by jimszalew** [comment]
explicitly saving items and orders helped to make them persist in the tests

File: app/views/orders/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <div class="col-md-3">
+      <%= @order.total_price %>
+    </div>
+    <div class="col-md-3">
+      <%= @order.created_at %>
+    </div>
+    <div class="col-md-3">
+      <% if @order.completed? || @order.cancelled? %>
+        <%= @order.resolved_at %>
+      <% end %>
```
Comment (MikelSage) [inline app/views/orders/show.html.erb:37]: nice use of conditional here

File: spec/factories/orders.rb [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
     user nil
+
+    trait :with_items do
+      transient do
+        item_count 3
+      end
+
+      after(:create) do |order, evaluator|
+        order.items << create_list(:item, evaluator.item_count)
+      end
```
Comment (MikelSage) [inline spec/factories/orders.rb:13]: Nice use of a trait here.

**Review by MikelSage — APPROVED** [review]
I think it looks good man

**Review by dphilla — APPROVED** [review]
This looks good! nice and and very explicit testing!

**Pull request merged**